geodocker-geoserver/Dockerfile.template

```diff
@@ -75,7 +75,37 @@ RUN set -x \
 RUN set -x \
     && rm /opt/tomcat/webapps/geoserver/WEB-INF/lib/guava-11.*.jar
 
+# Install Geomesa Kafka Distribution
+COPY --from=1 /geomesa-kafka* /opt/geomesa-kafka/
+
+# Install GeoMesa Kafka DataStore
+RUN set -x \
+  && cd /opt/tomcat/webapps/geoserver/WEB-INF/lib/ \
+  && tar zxvf /opt/geomesa-kafka/dist/gs-plugins/geomesa-kafka-${KAFKA_VERSION}-gs-plugin_2.11-${GEOMESA_VERSION}-install.tar.gz \
+  && rm /opt/tomcat/webapps/geoserver/WEB-INF/lib/zookeeper*.jar \
```

Comment:
is there already a ZK jar there?

Reply:
FYI if we can we should use ZK 3.4.10 instead of 3.4.6 everywhere due to bug fixes that have caused accumulo and kafka issues

Reply:
Yes, there already was a zookeeper jar there. When I run the `install-kafka.sh` script to add further dependencies, we ended up with multiple zookeeper jars of differing versions. That's why I deleted the zookeeper jar before running the script.

There's probably a better way to handle this (maybe handling this inside the `install-kafka.sh` script?), but this seemed to work

Reply:
hahaha prob always a better way but just wondering...might be worth a comment to mention it.

Reply:
Hmm, the `install-hadoop-accumulo.sh` script that gets run when installing the accumulo plugin installs zookeeper-3.4.9.jar, but `install-kafka.sh` script installs zookeeper-3.4.5.jar.

Maybe it's worth installing kafka plugin first, delete zookeeper-3.4.5.jar, then install accumulo plugin with zookeeper-3.4.9.jar